src/modm/board/module.md

```diff
@@ -72,11 +72,11 @@ This is great for just quickly prototyping something, however, when you want to
 use custom hardware, or even just change a few settings, it's better to use your
 own BSP:
 
-1. Generate the BSP closest to your custom hardware, then copy the files from
-   `modm/src/modm/board/{name}` to your own project and modify them.
-2. In your `project.xml` remove the board config inheritance (`<extends>`) and
+1. Generate the BSP closest to your custom hardware (using `lbuild build`).
+2. Copy the files from `modm/src/modm/board` to your own project and modify them.
+3. In your `project.xml` remove the board config inheritance (`<extends>`) and
```

Comment:
we should also clarify this while we're at it... if I figure out what it means I'll update the PR

Reply:
Since you're using a custom board config, you cannot extend the modm config anymore, or there would be conflicts. Technically you now need to manually merge the `board.xml` into your `project.xml`.

Reply:
And the obviously modify the config to your needs.

Reply:
The part I'm unsure about is "copy the pre-defined options into your own config". I clearly need to copy in the module dependencies, but then now I presumably need to do something to replace:

https://github.com/modm-io/modm/blob/22e9b6015c8d585b2dd116bdde6fd7acbfc8ea37/src/modm/board/disco_f429zi/module.lb#L40

And I am not sure what that is yet.

And then I need a `target` option set, which is listed in another repo (e.g. [here](https://github.com/modm-io/modm-devices/blob/develop/devices/stm32/stm32f4-01_11.xml)).

Reply:
Yes, that depends on your board. There's probably no predefined openoce config, you'll need to provide your own (see https://modm.io/reference/module/modm-build/#openocdsource).
You can also `lbuild discover -n :target` for the target, but you should already know the identifier.